src/config.ts

```diff
@@ -3,12 +3,8 @@ import env from 'env-var';
 export const config = {
```

Comment:
I think with the new changes you made, it does make sense to move the "config" inside the `core` folder and call it `env.ts` since it's only environment by now.

Also we do not need `env-var` anymore, it makes more sense to validate all env application-wise with the same technique, using zod like in modules.